app/home.html

```diff
@@ -8,7 +8,10 @@
     <link rel="stylesheet" type="text/css" href="./index-rtl.css" title="rtl" disabled>
   </head>
   <body>
-    <div id="app-content"></div>
+    <div id="app-content">
+      <img id="loading__logo" src="./images/logo/metamask-fox.svg" alt="" />
+      <img id="loading__spinner" src="./images/spinner.gif" alt="" />      
```

Comment:
Can we use classes here instead of ids if we are using the identifier to add styles?

```suggestion
      <img class="loading-logo" src="./images/logo/metamask-fox.svg" alt="MetaMask" />
      <img class="loading-spinner" src="./images/spinner.gif" alt="Loading" />      
```

Reply:
> Can we use classes here instead of ids if we are using the identifier to add styles?